test/test_helper.rb

```diff
@@ -12,24 +12,36 @@
 require 'helpers/resource_helper'
 require 'helpers/adapter_helper'
 
+require 'config/semian_test_config'
+
 Semian.logger = Logger.new(nil)
-Toxiproxy.populate([
-  {
-    name: 'semian_test_mysql',
-    upstream: 'localhost:3306',
-    listen: 'localhost:13306',
-  },
-  {
-    name: 'semian_test_redis',
-    upstream: 'localhost:6379',
-    listen: 'localhost:16379',
-  },
-  {
-    name: 'semian_test_net_http',
-    upstream: 'localhost:31050',
-    listen: 'localhost:31051',
-  },
-])
+
+class ToxiproxyConfig
```

Comment:
Running all this code in the body of the class is pretty sketchy. If we remove the nice helpers, we can move this to the top-level namespace as well.